src/components/List/Backdrop.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { ItemContainerConsumer } from "./Context";
+import { BackdropProvider } from "./BackdropContext";
+
+const KEYCODES = {
```

Comment:
I think you forgot to delete this and use the constant in `src/utils/keyCharCodes.js`

Reply:
This one is an old implementation of the Backdrop, which is tightly coupled with List, like Modal. I just moved it here, so I don't break anything

Reply:
Alright